tests/models/graph_tests.py

```diff
@@ -974,7 +974,7 @@ def test_derive_card_values(self):
             "http://www.cidoc-crm.org/cidoc-crm/P1_is_identified_by",
             graphid=graph.graphid,
         )
-        resource_graph.save()
+        resource_graph.save()  # `new()` doesn't persist to the database
```

Comment:
Not a blocker, but to me it looks like new() _does_ persist? Is it that save() does extra magic for related objects?

Reply:
lol not sure what I was thinking there. Ya will change 👍